wzdx/sample_files/raw/icone/get_icone.py

```diff
@@ -0,0 +1,27 @@
+import urllib.request
```

Comment:
Should this fetch be somewhere other than sample_files directory?

Reply:
I use this for validating iCone files with Ben, I will move it